SpliceAI.pm

```diff
@@ -268,11 +271,23 @@ sub run {
         $hash{'SpliceAI_cutoff'} = $result_flag;
       }
 
-      return ($self->{config}->{output_format} eq "json" || $self->{config}->{rest}) ?  {SpliceAI => \%hash} : \%hash;
+      $hash_aux{$data_value->{gene}} = \%hash;
     }
   }
 
-  return {};
+  return {} unless(%hash_aux);
+
+  my $result = {};
+  # Compare genes from SpliceAI with the variant gene
+  my $gene_symbol = $tva->transcript->{_gene_symbol} || $tva->transcript->{_gene_hgnc};
+  if($hash_aux{$gene_symbol}) {
+    $result = ($self->{config}->{output_format} eq "json" || $self->{config}->{rest}) ?  {SpliceAI => $hash_aux{$gene_symbol}} : $hash_aux{$gene_symbol};
+  }
+  else {
+    warn("Warning: for variant '$chr:$start-$end' the gene symbol '$gene_symbol' doesn't match any of the gene symbols from SpliceAI");
```

Comment:
could you please change Warning -> WARNING and add a newline to the end then the user wont be spammed with the line information.
And finally to make the warning a bit more clear I would suggest:
"WARNING: for variant position '$chr:$start-$end' the gene symbol '$gene_symbol' assigned by VEP doesn't match any of the gene symbols at the position in the SpliceAI file.\n"